SubmitHGCalPGun.py

```diff
@@ -29,6 +29,8 @@ def parseOptions():
     parser.add_option('', '--thresholdMax',  dest='thresholdMax',  type=float, default=35.0,    help='max. threshold value')
     parser.add_option('', '--etaMin',  dest='etaMin',  type=float, default=1.479,  help='min. eta value')
     parser.add_option('', '--etaMax',  dest='etaMax',  type=float, default=3.0,    help='max. eta value')
+    parser.add_option('', '--ptMin',  dest='ptMin',  type=float, default=-1.0, help='min. pt value, ignored when < 0, only used for physproc gun mode as phase space cut')
```

Comment:
This is a bit unfortunate, since we have `thresholdMin` and `thresholdMax` parameters as well that are used for the particle guns.

Reply:
True. Will reuse them.